.werft/vm/manifests.ts

```diff
@@ -74,6 +76,9 @@ spec:
               bootOrder: 1
               disk:
                 bus: scsi
+            - name: storage
+              disk:
+                bus: virtio
```

Comment:
I think this breaks VMs - I've reverted the one pushed to master here: https://github.com/gitpod-io/gitpod/pull/10735